https/README.md

```diff
@@ -15,17 +15,47 @@ The config file should be written in YAML format, and is reloaded on each connec
 
 ```
 tls_config:
-  # Certificate and key files for server to use to authenticate to client
+  # Certificate and key files for server to use to authenticate to client.
   cert_file: <filename>
   key_file: <filename>
 
-  # Server policy for client authentication. Maps to ClientAuth Policies
+  # Server policy for client authentication. Maps to ClientAuth Policies.
   # For more detail on clientAuth options: [ClientAuthType](https://golang.org/pkg/crypto/tls/#ClientAuthType)
   [ client_auth_type: <string> | default = "NoClientCert" ]
 
-  # CA certificate for client certificate authentication to the server
+  # CA certificate for client certificate authentication to the server.
   [ client_ca_file: <filename> ]
 
+  # Minimum TLS version that is acceptable.
+  [ min_version: <string> | default = "TLS1.2" ]
+
+  # Maximum TLS version that is acceptable.
+  [ max_version: <string> | default = "TLS1.3" ]
+
+  # List of supported cipher suites for TLS versions up to TLS 1.2. If empty,
+  # Go default cipher suites are used. Available cipher suites are documented
+  # below.
+  [ cipher_suites:
+    - <string> ] ]
+
+  # Disable HTTP/2 support. This can not be toggled on the fly.
+  [ disable_http2: <bool> | default = false ]
```

Comment:
We might using this configuration for things that aren't http over in the alertmanager, so it would be best not to mix http and tls in the same config section.

Reply:
done